types/github.com.hashicorp.nodeenrollment.types.v1.proto

```diff
@@ -51,6 +51,17 @@ message RootCertificates {
   google.protobuf.Struct state = 50;
 }
 
+// EncryptionKey contains the key information necessary to generate a sharedKey
+// NodeInformation and NodeCredentials will store their previous encryption key
+// using this message type
+message EncryptionKey{
+  bytes private_key_bytes = 1 [json_name="private_key_bytes"];
```

Comment:
This should be `private_key_pkcs8` to match nomenclature elsewhere and indicate how it's marshaled.